mobify/platform-scaffold
# AMP-89: pagination

**JIRA**: https://mobify.atlassian.net/browse/AMP-89

## Changes
- `<Pagination>` component 

## Todos
- [x] Waiting for PWA team to merge Pagination so can look if we can reuse any of parsers from PWA

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions PLP](http://localhost:3000/potions.html)
- Scroll down to bottom to check Pagination
